db/discord_db/db.py

```diff
@@ -0,0 +1,21 @@
+from base import Base
```

Comment:
order the imports :
inside packages 

third party packages 

your python files